aiomcache/client.py

```diff
@@ -134,44 +196,12 @@ def delete(self, conn, key):
         command = b'delete ' + key + b'\r\n'
         response = yield from self._execute_simple_command(conn, command)
 
-        if response not in (const.DELETED, const.NOT_FOUND):
-            raise ClientException('Memcached delete failed', response)
-        return response == const.DELETED
-
-    @acquire
-    def get(self, conn, key, default=None):
-        """Gets a single value from the server.
-
-        :param key: ``bytes``, is the key for the item being fetched
-        :param default: default value if there is no value.
-        :return: ``bytes``, is the data for this specified key.
-        """
-        values, _ = yield from self._multi_get(conn, key)
-        return values.get(key, default)
-
-    @acquire
-    def gets(self, conn, key, default=None):
-        """Gets a single value from the server together with the cas token.
-
-        :param key: ``bytes``, is the key for the item being fetched
-        :param default: default value if there is no value.
-        :return: ``bytes``, ``bytes tuple with the value and the cas
-        """
-        values, cas_tokens = yield from self._multi_get(
-            conn, key, with_cas=True)
-        return values.get(key, default), cas_tokens.get(key)
-
-    @acquire
-    def multi_get(self, conn, *keys):
-        """Takes a list of keys and returns a list of values.
+        if response == const.DELETED:
+            return True
+        elif response == const.NOT_FOUND:
```

Comment:
why original logic changed here?  `If` statement looks suspicious here without final `else`

Reply:
This is slightly faster, probably because it skips constructing the tuple of `(DELETED, NOT_FOUND)`. I could put the `raise` in an else block though.

Reply:
Lets return `raise` for some reason we follow this pattern here. I am not memcached expert, is it possible to get anything else then `DELETED` or `NOT_FOUND`?

Reply:
The `raise` is still there after the if/else block; the diff makes it not obvious because it's below a lot of removed lines.

I'm pretty sure those are the only possible responses to `delete`. But there could be subtle bugs outside of this library; for example, the process could fork, and with both parent and child processes sharing the connection, they could read each other's responses and thereby get a response code from a command other than `delete`, so the `raise` is still important.